waspc/e2e-test/ShellCommands.hs

```diff
@@ -129,3 +135,17 @@ dockerBuild :: ShellCommandBuilder ShellCommand
 dockerBuild =
   return
     "[ -z \"$WASP_E2E_TESTS_SKIP_DOCKER\" ] && cd .wasp/build && docker build . && cd ../.. || true"
+
+copyGitTrackedFilesFromRepoPath :: Path' (Rel ProjectRoot) (Dir source) -> ShellCommandBuilder ShellCommand
+copyGitTrackedFilesFromRepoPath repoRelDirPath = do
+  context <- ask
+  let sourceDir = toFilePath projectRootFromGoldenTestProjectDir </> toFilePath repoRelDirPath
+      destinationDir = "./" ++ _ctxtCurrentProjectName context
+
+      createDestinationDir = "mkdir -p " ++ destinationDir
+
+      listGitTrackedRepoRelDirFiles = "git -C " ++ toFilePath projectRootFromGoldenTestProjectDir ++ " ls-files " ++ toFilePath repoRelDirPath
+      filterRepoRelDirPath = "sed 's#^" ++ toFilePath repoRelDirPath ++ "##'"
+      copyFilesFromSourceToDestination = "rsync -a --files-from=- " ++ sourceDir ++ " " ++ destinationDir
+      copyRepoRelDirGitTrackedFiles = pipeShellCommands [listGitTrackedRepoRelDirFiles, filterRepoRelDirPath, copyFilesFromSourceToDestination]
```

Comment:
is `copyRepoRelDirGitTrackedFiles` the same thing as name of the whole function but just different ordering of the words? If so, can we somehow avoid that? Either skip having this varaible at all, or figure out how to rename it for real. But I think you can just skip it, and if you define piping operator, you can just use that in place below, it will be nice and more obvious.